src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -57,14 +57,25 @@ public enum LanguageVersion
         /// </list> 
         /// </remarks> 
         CSharp6 = 6,
+
         /// <summary>
         /// C# language version 7.
         /// </summary>
         CSharp7 = 7,
+
```

Comment:
or `700` how many minor releases are we expecting between majors?

_This takes be make to the old days when line number where required._